develop/dev-guide-sample-application-python.md

```diff
@@ -0,0 +1,1103 @@
+
+
+def trade_example() -> None:
+    with get_connection(autocommit=False) as conn:
+        with conn.cursor() as cur:
+            # create two players
+            # player 1: id is "1", has only 100 coins.
+            # player 2: id is "2", has 114514 coins, and 20 goods.
+            create_player(cur, ("1", 100, 0))
+            create_player(cur, ("2", 114514, 20))
+            conn.commit()
+
+        # player 1 wants to buy 10 goods from player 2.
+        # it will cost 500 coins, but player 1 cannot afford it.
+        # so this trade will fail, and nobody will lose their coins or goods
+        trade(conn, sell_id="2", buy_id="1", amount=10, price=500)
+
+        # then player 1 has to reduce the incoming quantity to 2.
+        # this trade will be successful
+        trade(conn, sell_id="2", buy_id="1", amount=2, price=100)
+
+        # let's take a look for player 1 and player 2 currently
+        with conn.cursor() as cur:
+            _, player1_coin, player1_goods = get_player(cur, "1")
+            print(f'id:1, coins:{player1_coin}, goods:{player1_goods}')
+            _, player2_coin, player2_goods = get_player(cur, "2")
+            print(f'id:2, coins:{player2_coin}, goods:{player2_goods}')
+
+
+simple_example()
+trade_example()
+```
+
+The driver has a lower level of encapsulation than ORM, so there are a lot of SQL statements in the program. Unlike ORM, there is no data object in drivers, so the `Player` queried by the driver is represented as a tuple.
+
+For more information about how to use mysql-connector-python, refer to [mysql-connector-python documentation](https://dev.mysql.com/doc/connector-python/en/).
+
+</div>
+
+</SimpleTab>
+
+## Step 3. Run the code
+
+The following content introduces how to run the code step by step.
+
+### Step 3.1 Table initialization
+
+Before running the code, you need to initialize the table manually. If you are using a local TiDB cluster, you can run the following command:
+
+<SimpleTab groupId="cli">
+
+<div label="MySQL CLI" value="mysql-client">
+
+```shell
+mysql --host 127.0.0.1 --port 4000 -u root < player_init.sql
+```
+
+</div>
+
+<div label="MyCLI" value="mycli">
+
+```shell
+mycli --host 127.0.0.1 --port 4000 -u root --no-warn < player_init.sql
+```
+
+</div>
+
+</SimpleTab>
+
+If you are not using a local cluster, or have not installed a MySQL client, connect to your cluster using your preferred method (such as Navicat, DBeaver, or other GUI tools) and run the SQL statements in the `player_init.sql` file.
+
+### Step 3.2 Modify parameters for TiDB Cloud
+
+If you are using a TiDB Cloud Serverless Tier cluster, you need to provide your CA root path and replace `<ca_path>` in the following examples with your CA path. To get the CA root path on your system, refer to [Where is the CA root path on my system?](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters#where-is-the-ca-root-path-on-my-system).
```

Comment:
This part is very different from the Chinese version. Need to have a check? https://github.com/pingcap/docs-cn/pull/12046/files#diff-01d806f55d31bf22f5076f9f113eb77498008b4be9fc174fd364668da961a3a3R828-R870

Reply:
PTAL. @Oreoxmt 
IMO, the current expression is better.
https://github.com/pingcap/docs/commit/7755bb060e22a827218800540ee2409b69c095a0

Reply:
@TomShawn IMO, maintaining two copies of the same content in the English doc might bring additional costs, such as the CA configuration in TiDB Cloud and Develop Guide. When there are changes needed in the https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters#where-is-the-ca-root-path-on-my-system, it would be easy to forget to update the same information in this doc. Therefore, I only reference the TiDB Cloud doc here.